### PR TITLE
rename the plugin repositories to match asdf-<tool>

### PR DESCRIPTION
**Pull request by johnlayton**

### CODE REVIEW

File: plugins/kubectx [hunk 1]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/johnlayton/asdf-kubectx.git
```
Comment (Stratus3D) [inline plugins/kubectx]: I don't see this plugin listed in the readme.

Comment (johnlayton) [reply plugins/kubectx]: Sorry. I noticed that this was to be provided by another user.

**Comment by johnlayton** [comment]
Yeah sorry, i noticed that someone else had a plugin for kubectx, i forgot the actual repo file

**Comment by Stratus3D** [comment]
Thanks for the PR!

**Pull request merged**